libraries/rush-lib/src/api/packageManager/PnpmPackageManager.ts

```diff
@@ -35,6 +43,10 @@ export class PnpmPackageManager extends PackageManager {
       this.pnpmfileFilename = RushConstants.pnpmfileV1Filename;
     }
 
+    this.subspacePnpmfileFilename = (subspaceName: string) => {
+      return `.pnpmfile-${subspaceName}.cjs`;
```

Comment:
The RFC said:
![image](https://github.com/microsoft/rushstack/assets/4673363/773e24cf-af37-43a1-8113-288b8cd086bc)

Reply:
fixed